plugins/knife/knife-clearwater-utils.rb

```diff
@@ -103,5 +103,14 @@ def continue? (prompt = "Continue?")
       end
     end
 
+    # Expands out hashes of boxes, e.g. {:bono => 3} becomes:
+    # {{:role => "bono", :index => 1}, {:role => "bono", :index = 2}, etc...
+    def create_cluster(boxes)
```

Comment:
Why is this in knife-clearwater-utils and not knife-deployment utils? (More generally, what are the criteria for choosing where to put new functions?)